core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -413,16 +413,18 @@ trait GenericSchema[R] extends SchemaDerivation[R] with TemporalSchema {
     val description: String = s"Either $typeAName or $typeBName"
 
     implicit val leftSchema: Schema[RA, A]  = new Schema[RA, A] {
-      override def nullable: Boolean                                         = true
-      override def canFail: Boolean                                          = evA.canFail
-      override def toType(isInput: Boolean, isSubscription: Boolean): __Type = evA.toType_(isInput, isSubscription)
-      override def resolve(value: A): Step[RA]                               = evA.resolve(value)
+      final override def nullable: Boolean                                         = true
```

Comment:
We could make this a `val` no?

Reply:
@kyri-petrou you wanna change that or shall we merge?

Reply:
Sorry guys missed the notifications on this one.

There's no real benefit in storing primitives that are simple expressions as vals. In fact, Scala vals are compiled to methods that point to private static fields, so likely more performant to have it as a `def` (if there's any difference at all)